academic-services/knowledge-exploration-service/MagDataToMakesJson.usql

```diff
@@ -0,0 +1,1228 @@
+    //////////////////////////////////////////////////////////////////////////////////////////////////
+    //                                                                                                
+    //  Copyright (c) Microsoft Corporation. All rights reserved.                                     
+    //  Licensed under the MIT License.                                                               
+    //                                                                                                
+    //////////////////////////////////////////////////////////////////////////////////////////////////
+
+    // Enables OUTPUT statements to generate dynamic files using column values
+    SET @@FeaturePreviews = "DataPartitionedOutput:on"; 
+    
+    ///////////////////////////////////////////////////////////////////////////////////////////////////
+    // Input Parameters                                                
+    //
+    // Input parameters that can be overridden by instantiating scripts                               
+    // See: https://docs.microsoft.com/en-us/u-sql/variables/declare-variables for details
+    ///////////////////////////////////////////////////////////////////////////////////////////////////
+    
+    // The Azure blob storage account name that contains the Microsoft Academic Graph data to be used by this script
+    DECLARE EXTERNAL @In_MagBlobAccount string = "";    // Example: "mymagstore"
+    
+    // The Azure blob storage container name that contains the Microsoft Academic Graph data to be used by this script
+    DECLARE EXTERNAL @In_MagBlobContainer string = "";  // Example: "mag-2020-02-07"
+    
+    // The Azure data lake storage output path where the JSON files will be created
+    DECLARE EXTERNAL @Out_OutputPath string = "";  // Example: "wasb://subgraphs@mymakesstore/mag-2020-02-07/microsoft"
```

Comment:
>subgraphs [](start = 71, length = 9)

Not consistent with the document, "makessubgraph" vs "subgraphs".
Please make them the same.